Makefile

```diff
@@ -15,6 +15,13 @@ clean:
 	git clone https://chromium.googlesource.com/external/gyp.git ./deps/gyp
 	cd deps/gyp && git checkout -q 0bb67471bca068996e15b56738fa4824dfa19de0
 
+# java / jni dependencies
+./deps/java:
+	mkdir -p ./deps/java/test
+	wget http://central.maven.org/maven2/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar -O ./deps/java/test/hamcrest-core-1.3.jar
+	wget http://central.maven.org/maven2/junit/junit/4.11/junit-4.11.jar -O ./deps/java/test/junit-4.11.jar
+	wget http://central.maven.org/maven2/com/google/code/findbugs/jsr305/3.0.0/jsr305-3.0.0.jar -O ./deps/java/jsr305-3.0.0.jar
+
```

Comment:
So wget doesn't usually exist on OS X. Switching to curl (and https) would be an option, but for the sake of keeping the initial build simple I'd rather just keep these in the repo.